libraries/AP_Gripper/AP_Gripper_Servo.cpp

```diff
@@ -32,10 +28,6 @@ void AP_Gripper_Servo::release()
     // move the servo to the release position
     SRV_Channels::set_output_pwm(SRV_Channel::k_gripper, config.release_pwm);
```

Comment:
I think we need a:

config.state = AP_Gripper::STATE_RELEASING;

Reply:
Thanks Peter, that is a stupid mistake. Thanks for pointing it out!